sdk/identity/identity/src/credentials/azurePipelinesCredential.ts

```diff
@@ -72,19 +69,16 @@ export class AzurePipelinesCredential implements TokenCredential {
    */
   public async getToken(
     scopes: string | string[],
-    options?: GetTokenOptions,
+    options?: GetTokenOptions
   ): Promise<AccessToken> {
     if (!this.clientAssertionCredential) {
-      const errorMessage = `${credentialName}: is unavailable. To use Federation Identity in Azure Pipelines, these are required as input parameters / env variables - 
-      tenantId,
-      clientId,
-      serviceConnectionId,
-      "SYSTEM_TEAMFOUNDATIONCOLLECTIONURI" &&
-      "SYSTEM_TEAMPROJECTID" &&
-      "SYSTEM_PLANID" &&
-      "SYSTEM_JOBID" &&
+      const errorMessage = `${credentialName}: is unavailable. To use Federation Identity in Azure Pipelines, the "systemAccessToken" is required as input parameter and should be set as part of the devops task as the environment variable.Additionally, following should be provided either as optional parameters OR env variables - 
+      tenantId OR "AZURESUBSCRIPTION_TENANT_ID",
```

Comment:
Should the error here suggest using one of the supported tasks instead of the variable names? The `AZURESUBSCRIPTION*` values aren't ever set by the user; they're set by the infrastructure on the golden path.

Reply:
the TSG could explain the supported task list, etc.

Reply:
I added in the examples section